io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: io.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: cpu-x
+rename-icon: com.github.thetumultuousunicornofdarkness.cpu-x
+rename-desktop-file: com.github.thetumultuousunicornofdarkness.cpu-x.desktop
+rename-appdata-file: com.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # OpenGL/Vulkan
+  - --device=dri
+  # Needs use of 'pkexec' to run 'cpu-x-daemon'
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - gtkmm.json
+
+  - name: libcpuid
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_DOCS=0
+      - -DBUILD_SHARED_LIBS=0
+    sources:
+      - type: git
+        url: https://github.com/anrieff/libcpuid.git
+        tag: "v0.6.4"
+        commit: 02237e67f5cbff348c63ad5338a70d75909a23b1
+    cleanup:
+      - /bin
+      - /include
+      - '*.a'
+      - /lib/cmake
+      - /lib/pkgconfig
+
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS=0
+      - -DGLFW_BUILD_TESTS=0
+      - -DGLFW_BUILD_DOCS=0
+      - -DGLFW_USE_WAYLAND=1
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 3fa2360720eeba1964df3c0ecf4b5df8648a8e52
+      - type: patch
+        path: patches/0001-Platform-Prefer-Wayland-over-X11.patch
+    cleanup:
+      - /include
+      - '*.a'
+      - /lib/cmake
+      - /lib/pkgconfig
+
+  - name: cpu-x
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DFLATPAK=1
+    sources:
+      - type: git
+        url: https://github.com/TheTumultuousUnicornOfDarkness/CPU-X.git
+        tag: "v5.0.1"
```

Comment:
You didn't update the tag/commit here to include the launchable change in appdata https://github.com/TheTumultuousUnicornOfDarkness/CPU-X/commit/1c1d5079a266f539f40535b6e37723ac05dc74cc

This commit is still wrong the id should be `io.github` not `com.github`

Instead of coplicating everything with patches + appstream-util commands, fix all these upstream, make a new tag

Reply:
OK, fine, I updated AppID uptream: https://github.com/TheTumultuousUnicornOfDarkness/CPU-X/commit/da2bd23203856f2b308a7cf30fa0c53c4276003c

AppData looks good locally:
```
$ flatpak run --command=appstream-util org.flatpak.Builder validate /home/vagrant/.local/share/flatpak/app/io.github.thetumultuousunicornofdarkness.cpu-x/x86_64/master/85db40e132e6d7bb01a8933403cf8cc94178db2b9a3b9262b2d1f7b8273a4732/export/share/metainfo/io.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml
/home/vagrant/.local/share/flatpak/app/io.github.thetumultuousunicornofdarkness.cpu-x/x86_64/master/85db40e132e6d7bb01a8933403cf8cc94178db2b9a3b9262b2d1f7b8273a4732/export/share/metainfo/io.github.thetumultuousunicornofdarkness.cpu-x.appdata.xml: OK
```